### PR TITLE
Add console.flag extension for kubevirt dynamic plugin

### PR DESCRIPTION
**Pull request by glekner**
## 📝 Description

Add console.flag extension for 4.11 Cluster.

If plugin is installed, we will add: `"disallowed": ["KUBEVIRT_DYNAMIC"]`
to the static extensions that we are upgrading with the dynamic ones that we currently have

### CODE REVIEW

**Comment by glekner** [comment]
@yaacov please review

File: locales/en/plugin__kubevirt-plugin.json [hunk 1]
```diff
@@ -439,5 +439,8 @@
   "MAC Address": "MAC Address",
   "masquerade": "masquerade",
   "VMI": "VMI",
-  "Virtual Machines Instances": "Virtual Machines Instances"
+  "Virtual Machines Instances": "Virtual Machines Instances",
```
Comment (glekner) [inline locales/en/plugin__kubevirt-plugin.json:442]: turns out we are missing some extractions in `main`.
hijacking this PR to fix

Comment (yaacov) [reply locales/en/plugin__kubevirt-plugin.json:442]: should be `VirtualMachinesInstances` it's a CRD name.

**Pull request merged**